package.json

```diff
@@ -64,6 +65,7 @@
     "err-code": "^1.1.2",
     "interface-datastore": "~0.7.0",
     "ipfs-block": "~0.8.1",
+    "ipfs-repo-migrations": "ipfs/js-ipfs-repo-migrations#master",
```

Comment:
This can point to the released version now, correct?

Reply:
Yes, I still have not released it as I want to have LGTM for the PRs in case of some changes would have to be made. It will be the last step before merge.